tsdb/mapper.go

```diff
@@ -306,7 +338,7 @@ func (lm *LocalMapper) nextChunkRaw() (*MapperOutput, error) {
 // for the current tagset. Tagsets are always processed in the same order as that
 // returned by AvailTagsSets(). When there is no more data for any tagset nil
 // is returned.
-func (lm *LocalMapper) nextChunkAgg() (*MapperOutput, error) {
+func (lm *LocalMapper) nextChunkAgg() (interface{}, error) {
```

Comment:
Yeah, at the time it returned an interface, but the only caller just reflected it back to a MapperOutput so it was just doing unnecessary work.  If it needs to be more flexible now that's fine.